syntax = "proto3";
package tr451_vomci_function_sbi_service;

import "google/protobuf/empty.proto";
import "tr451_vomci_function_sbi_message.proto";

service OmciFunctionHelloSbi {
    rpc HelloVomci (tr451_vomci_function_sbi_message.HelloVomciRequest) returns (tr451_vomci_function_sbi_message.HelloVomciResponse);
}

service OmciFunctionMessageSbi {
    rpc ListenForOmciRx (google.protobuf.Empty) returns (stream tr451_vomci_function_sbi_message.OmciPacket);
    rpc OmciTx (tr451_vomci_function_sbi_message.OmciPacket) returns (google.protobuf.Empty);
}
